src/Mod/Fem/femsolver/calculix/write_femelement_material.py

```diff
@@ -48,6 +48,9 @@ def is_density_needed():
             return True
         return False
 
+    f.write("** Physical constants for SI(mm) unit system with Kelvins\n")
```

Comment:
Thank you for noticing! A quick fix would be:
```suggestion
    f.write("\n** Physical constants for SI(mm) unit system with Kelvins\n")
```
The generated .inp files currently look like this (material_multiple_bendingbeam_fiveboxes.inp) [notice no new line after the NSET card and the "physical constants" comment]:
```
*NSET, ...
...
4152,
4153,
** Physical constants for SI(mm) unit system with Kelvins
*PHYSICAL CONSTANTS, ABSOLUTE ZERO=0, STEFAN BOLTZMANN=5.670374419e-11
```

Reply:
@marioalexis84 can you take a look?

Reply:
@chennes, it's done.

Reply:
Once the diff checker is fixed, the tests will fail because the generated .inp will have a new line before `** Physical constants ...`. This is the only file where a new line needs to be added, the rest of them are fine
https://github.com/FreeCAD/FreeCAD/blob/563b1cbc286ed6ab1bea6ba28358f61095eb0da4/src/Mod/Fem/femtest/data/calculix/box_frequency.inp#L429-L430

Reply:
Yes, that's the only one. Now all .inp files have a new line before the physical constants line.